catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/TestMultipleJDBCLoad.java

```diff
@@ -72,33 +76,26 @@ public static void startup() throws IOException {
           DOWNLOAD_JDBC_DRIVER_URL, pgDirPath.toString(), icebergLibsPath.toString());
     }
 
-    mySQLContainer =
-        new MySQLContainer<>(MYSQL_DEFAULT_IMAGE_NAME)
-            .withDatabaseName(TEST_DB_NAME)
-            .withUsername("root")
-            .withPassword("root");
-    mySQLContainer.start();
-    postgreSQLContainer =
-        new PostgreSQLContainer<>(CatalogPostgreSqlIT.DEFAULT_POSTGRES_IMAGE)
-            .withDatabaseName(TEST_DB_NAME)
-            .withUsername("root")
-            .withPassword("root");
-    postgreSQLContainer.start();
+    containerSuite.startMySQLContainer(TEST_DB_NAME);
+    mySQLContainer = containerSuite.getMySQLContainer();
+    containerSuite.startPostgreSQLContainer(TEST_DB_NAME);
+    postgreSQLContainer = containerSuite.getPostgreSQLContainer();
   }
 
   @Test
-  public void testCreateMultipleJdbc() throws URISyntaxException {
+  public void testCreateMultipleJdbc() throws URISyntaxException, SQLException {
     String metalakeName = RandomNameUtils.genRandomName("it_metalake");
     String postgreSqlCatalogName = RandomNameUtils.genRandomName("it_postgresql");
     GravitinoMetalake metalake =
         client.createMetalake(NameIdentifier.of(metalakeName), "comment", Collections.emptyMap());
 
     Map<String, String> pgConf = Maps.newHashMap();
-    String jdbcUrl = postgreSQLContainer.getJdbcUrl();
+    String jdbcUrl = postgreSQLContainer.getJdbcUrl(TEST_DB_NAME);
     String database = new URI(jdbcUrl.substring(jdbcUrl.lastIndexOf("/") + 1)).getPath();
```

Comment:
Are TEST_DB_NAME and database the same?

Reply:
They are the same, I'll assign `TEST_DB_NAME.toString()` to this.database directly.